overview.html

```diff
@@ -523,6 +524,89 @@ <h2 id="J2EE">J2EE</h2>
     configuration of your JNDI provider to register this factory and reference this library.
 </P>
 
+<h2 id="Logging">Logging</h2>
+
+<P>
+    This library uses the {@link java.util.logging} package. Classes that log will register a
+    {@link java.util.logging.Logger} under their fully qualified class name.
+    By default java-cloudant does not apply any logging configuration so the system defaults will be
+    used.
+</P>
+
+<P>
+    There are two ways to configure {@link java.util.logging}. It can be configured via a properties
+    file or programmatically. These examples show configuration for writing to the console for FINE
+    level or more severe for all {@code com.cloudant*} classes. The JVM documentation contains more
+    information about {@link java.util.logging} and many other logging libraries are also able to
+    integrate with it.
+</P>
+
+<OL>
+    <LI>Configuring using a properties file
+        <pre>
+            {@code
+            # "handlers" specifies a comma separated list of log Handler
+            # classes. For this example it will use only the Console Handler
+            # however there a other handlers such as FileHandler
+            # see http://docs.oracle.com/javase/7/docs/api/java/util/logging/package-summary.html
+            # for a list of handlers provided by the JDK.
+            handlers=java.util.logging.ConsoleHandler
+
+            # Limit the messages that are printed on the console to FINE and above.
+            # Full list of levels http://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html
+            java.util.logging.ConsoleHandler.level=FINE
+
+            # Set the formatter for this handler, there are several default formatter classes
+            # available, or you can create your own custom formatter.
+            java.util.logging.ConsoleHandler.formatter=java.util.logging.SimpleFormatter
+
+            # Set cloudant sync classes to log at level FINE
+            com.cloudant.level=FINE
```

Comment:
No, you need both.
Basically the handler determines what the handler will print, whilst the one on the logger determines which levels the logger will send to its handlers.
In this way it is possible to have a console handler with level `SEVERE`, but a file handler with level `INFO` and if both those handlers were registered to the com.cloudant logger then with `com.cloudant.level=INFO` you would get INFO messages in the file, but only SEVERE messages on the console. If you set `com.cloudant.level=SEVERE` then you would only get severe in both the console and the file.

Does that make sense?